src/NumberGuessingGame.js

```diff
@@ -1,77 +1,63 @@
-import React, { Component } from "react";
+import React, { useState } from "react";
 import GuessControl from "./GuessControl";
 import GuessMessage from "./GuessMessage";
 import GameOver from "./GameOver";
 
-/**
- *
- * Returns a random integer number from 1-100 inclusive
- */
-function getRandomNumber() {
-  return Math.floor(Math.random() * 100) + 1;
-}
 
-const MAX_ATTEMPTS = 5;
+function NumberGuessingGame() {
 
-class NumberGuessingGame extends Component {
-  constructor(props) {
-    super(props);
-
-    this.state = {
+  const [ numberData, setNumberData ] = useState({
```

Comment:
We should place each of these values in separate useStates. No need to force them in an object.